recipe/meta.yaml

```diff
@@ -163,42 +165,43 @@ outputs:
         - {{ pin_compatible('numpy') }}
         - packaging >=23.2
         - panda-client >=1.5.70
-        - pandas >=2.1.4,<3
+        - pandas >=2.1.4
         - parsl ==2024.4.29
         - patch >=2.7.6
         - pgcli >=3.5.0
         - photutils >=1.8.0
-        - piff >=1.3.3
+        - piff >=1.5
         - prmon >=3.0.2  # [linux]
         - psycopg2 >=2.9.9,<3
         - pyarrow >=14.0.2
         - pybind11 >=2.12.0
         - pydantic >=2.7.1,<3
         - pyld >=2.0.3
-        - pysynphot >=2.0.0
         - pytables >=3.9.2
-        - python >=3.11,<3.12
-        - python-confluent-kafka >=1.9.2,<2
-        - pytorch >=2.1.0
+        - python >=3.12,<3.13
+        - python-confluent-kafka >=1.9.2
+        - pytorch >=2.1.0   # [not aarch64]
+        - pytorch >=2.1.0,<=2.3.1  # [aarch64]
         - pyyaml >=6.0.1
+        - qp
         - requests >=2.31.0
         - rucio-clients >=34.3.0
-        - ruff ==0.5.2
+        - ruff >=0.5.2
         - s3fs >=2024.6.1
         - schwimmbad >=0.3.2
         - scikit-image >=0.22.0
         - scikit-learn >=1.3.2
         - scipy >=1.13.0,<1.14
         - seaborn >=0.13.0
-        - skyproj >=1.2.3
+        - skyproj >=2.0.0
         - sorcha >=0.9.1
         - spherematch >=0.10.1
         - sqlalchemy >=2.0.23
         - sqlite >=3.44.2
         - threadpoolctl >=3.2.0
         - torchvision >=0.16.1
         - tqdm >=4.66.1
-        - treecorr >=4.3.3,<5
+        - treecorr >=4.3.3
```

Comment:
We've been given assurances of compatibility, right?  Are we sure we don't need a minimum of 5 now?

Reply:
This should be `treecorr >=5.0.2,<6` I think.